src/app/model/Organization.ts

```diff
@@ -4,7 +4,7 @@ import { Resource } from "./schema";
 import { createContext, useContext } from "react";
 
 const defaultData: OrganizationInterface = {
-  uid: "",
+  uid: "-1",
```

Comment:
There was a weird error that would break the site if this was just an empty string. initializing this just incase